app/views/manage/users/show.html.haml

```diff
@@ -2,4 +2,20 @@
 
 %h1= @user.name
 %h2= @user.email
-  
\ No newline at end of file
+
+%h3 Your Indices:
+%table.table.table-condensed.table-striped
+  %thead
+    %tr
+      %th Community
+      %th Show Index
+  %tbody
+    - Index.user_indices(current_user).each do |index|
```

Comment:
Better method for this is to pull these out into an instance variable in the manager/users controller